notebooks/vertex_genai/labs/vertex_llm_evaluation.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    parameters = None  # TODO
</pre><p> </p><p>Maybe just a matter of preference, but I prefer using three dots operator <code>...</code> , like <code>parameters = ...</code> which explicitly tells this code is not complete, instead of <code>None</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/445/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I think the "..." triggers issues with pre-commit, while "None" is okay because regular Python syntax.</p>

Reply:
`...` is a regular Python expression, so ok for pre-commit as well.